runtest.py

```diff
@@ -88,10 +90,17 @@ def clean_up_output_data(self):
             shutil.rmtree(output_dir)
 
     def _load_settings(self):
+        if not self.firefox_settings:
+            return {}
         try:
             if os.path.exists(self.firefox_settings):
                 with open(self.firefox_settings) as firefox_settings_fh:
-                    return json.load(firefox_settings_fh)
+                    settings = json.load(firefox_settings_fh)
+                    self.logger.info('** Important **\n'
```

Comment:
I skipped '\n' part because I wasn't writing the code, but adding a '\n' is totally fine. I just want to make sure people see the warning clearly before they get back to complain this.